privacy-protections/surrogates/main.js

```diff
@@ -0,0 +1,77 @@
+function loadSurrogates () {
+
+    for (const [name, testData] of Object.entries(surrogates)) {
+        const s = document.createElement('script')
+        testData.crossOrigin ? s.crossOrigin = testData.crossOrigin : ''
+        s.src = testData.url
+        
+        s.onload = () => updateTable({name, testData})
+        s.onerror = (error) => updateTable({name, testData, error})
+
+        document.body.appendChild(s)
+    }
+}   
+
+function updateTable ({name, testData, error}) {
```

Comment:
can we also have a global object with test results like in other tests? This way, when testing this page automatically, you will not need to scrape HTML, you'll just pull info from `window.results`.